api/src/main/java/org/apache/gravitino/authorization/SecurableObjects.java

```diff
@@ -88,6 +88,22 @@ public static SecurableObject ofTable(
     return of(MetadataObject.Type.TABLE, names, privileges);
   }
 
+  /**
```

Comment:
Column isn't the securable object. So you should remove this.

Reply:
@jerqi ok, done.